### ISSUE TITLE
No Application Seems to be Running Here error

### ISSUE THREAD

**Issue by kmishra9** [shadowcodex/c9.ide.desktop#4]
Hey, wondering what I should do if I'm getting this screen? All I've done thus far is follow the 2 instructions given to install and enable plugins. Haven't started a server, haven't done anything at all, I'm just not sure why I'm getting this error (never occurred on a different workspace that someone had shared with me).

If its an installation error, how would I go about reinstalling? 
<img width="960" alt="cloud9 issue" src="https://cloud.githubusercontent.com/assets/10662252/15854330/26f34b96-2c5e-11e6-919d-3634b79ec805.PNG">

**Comment by shadowcodex** [comment]
Probably means the desktop isn't actually started. 

try View>Restart and then View>Open Desktop

**Comment by kmishra9** [comment]
Hmm, so I actually don't have a restart button anywhere in View I have a "Start desktop" and "Open Desktop." And yes, I've tried starting the desktop several times, refreshing and restarting C9, etc. to fix the issue to no avail. A friend I contacted who had previously used this also ran into this same error at some point, but was able to resolve it by reinstalling the plugin.

How would I go about uninstallling the currently broken plugin, so that I can reinstall it? Or, better yet, if you'd like I can share the workspace with you, if you'd like to examine the issue with more detail.
